pandas/formats/format.py

```diff
@@ -2138,6 +2016,31 @@ def formatter(value):
 
         return formatter
 
+    def _get_formatted_data(self):
+        """
+        Returns formatted float values. This is different from
```

Comment:
Ah, I see.  Fair enough, then a note should be made about that in the module doc-string.  I'll move this functionality then into `get_result_as_array` then.